Managing projects

clample opened the pull request.

Commit:
Add initial managing your apps documentation

Document app preperation, creating a new project, managing app versions

Commit:
Remove duplicated info from Account page

We now cover uploading an app in "Managing Your Apps". Account should
only be concerned with things such as managing team, plans, billing.

Commit:
Add image for app version icons

ragog commented on docs/general-reference/managing-your-apps.md:
```diff
@@ -0,0 +1,53 @@
+
+<h3 id="create-new-project">Create a New Project</h3>
+
+If this isn't the first project on your account, you will first need to select the "New App" button from your home screen:
```
I think most people looking at this guide might be in the other situation: nothing uploaded and they want some info on how to proceed (which option to select). Let's give more weight to this scenario, while still mentioning the second to make sure people can follow.

ragog commented on docs/general-reference/managing-your-apps.md:
```diff
@@ -0,0 +1,53 @@
+<img class="center shadow" src="/img/dashboard/app-dashboard.png">
+
+
+Clicking "All Versions" gives you an overview of all previously uploaded versions of this app. On the left side of each app version, you see the ID. This can be used to specify specific app versions when performing automated testing. If you don't specify an app version, the active app version will be used. Also, for manual testing, the active app version is always used. You can see which version  ...
```
It's small stuff, but "specify specific" sounds awful ;)

Commit:
Change wording

Commit:
Attempt to clarify for first time users

First time users will not need to select the New App button. We can
mention this explicitly so that the guide is easier for them to follow.

Commit:
Reorganise "Create a New Project" section

It is more natural to show how to make your first project, and then how
to make additional projects.

Commit:
Merge branch 'master' of https://github.com/testobject/help into managing-projects
